Resources/doc/cookbook/recipe_sortable_listing.rst

```diff
@@ -49,13 +49,13 @@ Then we need to inject the Sortable listener. If you only have the Gedmo bundle
 
 .. code-block:: yaml
 
-	services:
-        gedmo.listener.sortable:
+    services:
+        gedmo.listener.sortable: true
```

Comment:
Why we need this `true`? I doesn't see it in [gedmo documentation](https://github.com/Atlantic18/DoctrineExtensions/blob/master/doc/symfony2.md)